webapp/src/components/App/Studies/LauncherDialog.tsx

```diff
@@ -182,7 +195,7 @@ function LauncherDialog(props: Props) {
    */
   const parseHoursToSeconds = (hourString: string): number => {
     const seconds = moment.duration(hourString, "hours").asSeconds();
-    return seconds > 0 ? seconds : DEFAULT_TIME_LIMIT;
+    return Math.max(minSeconds, Math.min(seconds, maxSeconds));
```

Comment:
⚠️ ⚠️ `return Math.max(minSeconds, Math.min(seconds, maxSeconds));` logical error => this will always return 3600, and so entirely block this field

Reply:
Non, je n'ai pas constaté ce problème lorsque j'ai testé.

Par exemple, en Python, ça fonctionne très bien :

```python
minSeconds = 3600
maxSeconds = 240 * 3600


def parse_hours_to_seconds(hour_string: str) -> int:
    seconds = int(hour_string) * 3600
    return max(minSeconds, min(seconds, maxSeconds))


assert parse_hours_to_seconds("0") == minSeconds
assert parse_hours_to_seconds("1") == minSeconds
assert parse_hours_to_seconds("123") == 123 * 3600
assert parse_hours_to_seconds("240") == maxSeconds
assert parse_hours_to_seconds("241") == maxSeconds
```

Reply:
Oui en fait cela vient de l'api qui retourne 3600 pour le launcher local, mis a part cela la synthaxe est bonne mais niveau implementation c'est quelque chose a eviter de forcer la validation comme cela